stdlib/_winapi.pyi

```diff
@@ -1,123 +1,123 @@
-    PAGE_NOACCESS: Literal[0x1]
-    PAGE_NOCACHE: Literal[0x200]
-    PAGE_READONLY: Literal[0x2]
-    PAGE_READWRITE: Literal[0x4]
-    PAGE_WRITECOMBINE: Literal[0x400]
-    PAGE_WRITECOPY: Literal[0x8]
-
-    PROCESS_ALL_ACCESS: Literal[0x1FFFFF]
-    PROCESS_DUP_HANDLE: Literal[0x40]
-
-    SEC_COMMIT: Literal[0x8000000]
-    SEC_IMAGE: Literal[0x1000000]
-    SEC_LARGE_PAGES: Literal[0x80000000]
-    SEC_NOCACHE: Literal[0x10000000]
-    SEC_RESERVE: Literal[0x4000000]
-    SEC_WRITECOMBINE: Literal[0x40000000]
-
-    STARTF_USESHOWWINDOW: Literal[0x1]
-    STARTF_USESTDHANDLES: Literal[0x100]
-
-    STD_ERROR_HANDLE: Literal[0xFFFFFFF4]
-    STD_OUTPUT_HANDLE: Literal[0xFFFFFFF5]
-    STD_INPUT_HANDLE: Literal[0xFFFFFFF6]
-
-    STILL_ACTIVE: Literal[259]
-    SW_HIDE: Literal[0]
-    SYNCHRONIZE: Literal[0x100000]
-    WAIT_ABANDONED_0: Literal[128]
-    WAIT_OBJECT_0: Literal[0]
-    WAIT_TIMEOUT: Literal[258]
+    INVALID_HANDLE_VALUE: Final = 0xFFFFFFFFFFFFFFFF  # noqa: Y054
+    IDLE_PRIORITY_CLASS: Final = 0x40
+    NORMAL_PRIORITY_CLASS: Final = 0x20
+    REALTIME_PRIORITY_CLASS: Final = 0x100
+    NMPWAIT_WAIT_FOREVER: Final = 0xFFFFFFFF
+
+    MEM_COMMIT: Final = 0x1000
+    MEM_FREE: Final = 0x10000
+    MEM_IMAGE: Final = 0x1000000
+    MEM_MAPPED: Final = 0x40000
+    MEM_PRIVATE: Final = 0x20000
+    MEM_RESERVE: Final = 0x2000
+
+    NULL: Final = 0
+    OPEN_EXISTING: Final = 3
+
+    PIPE_ACCESS_DUPLEX: Final = 3
+    PIPE_ACCESS_INBOUND: Final = 1
+    PIPE_READMODE_MESSAGE: Final = 2
+    PIPE_TYPE_MESSAGE: Final = 4
+    PIPE_UNLIMITED_INSTANCES: Final = 255
+    PIPE_WAIT: Final = 0
+
+    PAGE_EXECUTE: Final = 0x10
+    PAGE_EXECUTE_READ: Final = 0x20
+    PAGE_EXECUTE_READWRITE: Final = 0x40
+    PAGE_EXECUTE_WRITECOPY: Final = 0x80
+    PAGE_GUARD: Final = 0x100
+    PAGE_NOACCESS: Final = 0x1
+    PAGE_NOCACHE: Final = 0x200
+    PAGE_READONLY: Final = 0x2
+    PAGE_READWRITE: Final = 0x4
+    PAGE_WRITECOMBINE: Final = 0x400
+    PAGE_WRITECOPY: Final = 0x8
+
+    PROCESS_ALL_ACCESS: Final = 0x1FFFFF
+    PROCESS_DUP_HANDLE: Final = 0x40
+
+    SEC_COMMIT: Final = 0x8000000
+    SEC_IMAGE: Final = 0x1000000
+    SEC_LARGE_PAGES: Final = 0x80000000
+    SEC_NOCACHE: Final = 0x10000000
+    SEC_RESERVE: Final = 0x4000000
+    SEC_WRITECOMBINE: Final = 0x40000000
+
+    STARTF_USESHOWWINDOW: Final = 0x1
+    STARTF_USESTDHANDLES: Final = 0x100
+
+    STD_ERROR_HANDLE: Final = 0xFFFFFFF4
+    STD_OUTPUT_HANDLE: Final = 0xFFFFFFF5
+    STD_INPUT_HANDLE: Final = 0xFFFFFFF6
+
+    STILL_ACTIVE: Final = 259
+    SW_HIDE: Final = 0
+    SYNCHRONIZE: Final = 0x100000
+    WAIT_ABANDONED_0: Final = 128
+    WAIT_OBJECT_0: Final = 0
+    WAIT_TIMEOUT: Final = 258
 
     if sys.version_info >= (3, 10):
         LOCALE_NAME_INVARIANT: str
         LOCALE_NAME_MAX_LENGTH: int
         LOCALE_NAME_SYSTEM_DEFAULT: str
-        LOCALE_NAME_USER_DEFAULT: str | None
+        LOCALE_NAME_USER_DEFAULT: Final[str | None]
```

Comment:
I'd leave this one as-is because it doesn't fit the pattern of the other changes being proposed in this PR. Let's aim for a clean separation between this PR and the planned followup, so it's easy when looking through `git blame` later exactly what kinds of changes occurred in the first PR and exactly waht kinds of changes occurred in the second PR.

Reply:
Thanks, I'm trying to remember if I accidentally changed this, or why, good catch: https://github.com/python/typeshed/pull/12332/commits/f981c303f5254d18718076448fb5d4d41c36599d